app/models/physical_server_provision_request.rb

```diff
@@ -11,7 +11,7 @@ def src_configured_systems
   end
 
   def self.request_task_class_from(_attribs)
-    ManageIQ::Providers::Lenovo::PhysicalInfraManager::ProvisionTask
+    PhysicalServerProvisionTask
```

Comment:
Does this method need to be overridden in sub-classes?

Reply:
Okay so as discussed on a call a week ago we're now allowing Task override from within provider repo (see L19 below). Overriden Task should inherit from this `PhysicalServerProvisionTask`.

Here in Request, however, we cannot allow override because there is no way we can figure what provider does the Request belong to. But this is not a problem, because `self.request_task_class` is only used by Request to render Task description prefix, so IMO it's safe to leave it like this.